extensions/amp-accordion/1.0/test/test-accordion.js

```diff
@@ -73,7 +73,7 @@ describes.sandboxed('Accordion preact component', {}, (env) => {
       const content = dom.children[1];
       expect(content.localName).to.equal('div');
       expect(content.innerHTML).to.equal('content1');
-      expect(content.hidden).to.be.false;
+      expect(content.getAttribute('aria-hidden')).to.equal('false');
```

Comment:
Looks like these lines have to be removed as well

Reply:
Yep latest commit has updates to tests which includes this line.